File: src/mutable_marker/mutable_marker.erl
-module(mutable_marker).

% elp:ignore W0012 W0040 E1599
-moduledoc {file, "../../docs/mutable_marker.md"}.

-export([
    run/0
]).

run() ->
    temporal_sdk:start_workflow(cluster_1, "default", mutable_marker_workflow, [
        {workflow_task_timeout, 1_000}
    ]),
    ok.

File: lib/mutable_marker/mutable_marker.ex
defmodule MutableMarker do
  @external_resource "docs/mutable_marker.md"
  @moduledoc TemporalSdk.Utils.Code.exdoc!("docs/mutable_marker.md")

  def run do
    TemporalSdk.start_workflow(:cluster_1, "default", MutableMarker.Workflow,
      workflow_task_timeout: 1_000
    )

    :ok
  end
end
